menu.csv

```diff
@@ -0,0 +1,12 @@
+Wild Scallop Crudo,"beets, horseradish, olive oil",17,Daily
+Black Winter Truffle Ramen,"Périgord truffles, egg yolk, Hozon",48,Daily
+Smothered Katsu Chicken,"pickled green beans, shiitake gravy",24,Daily
+Brussels Sprouts,"miso, chili, pine nuts",14,Daily
+Shrimp,"spicy mayo, pickled red onion, iceberg",13,Buns
+Shiitake,"hoisin, scallion, cucumber",13,Buns
+Chicken Meatball,"jalapeño, iceberg, paprika mayo",13,Buns
+Brisket,"horseradish, pickled red onion, cucumber",13,Buns
+Swiss Chard,"Asian pear, sesame, kimchi vinaigrette",8,Small Plates
+Spicy Cucumber,"scallion, togarashi, almonds",8,Small Plates
+Smoked Chicken Wings,"pickled chili, garlic, scallion",15,Small Plates
+BBQ Pork Belly Ssäm,"lettuce cups, kimchi",17,Small Plates
```

Comment:
Thanks! Had a glance on Wikipedia - looking forward to learn how to work with it =)